qgreenland/cli/run.py

```diff
@@ -72,11 +72,4 @@ def run(pattern, dry_run, fetch_only, workers) -> None:
     )
 
     if not result.scheduling_succeeded:
-        raise click.UsageError(
```

Comment:
This was useful when this mistake was possible. Now the `cli.sh` script checks if you're trying to do a "run" and prints a message like this, directing you to use `container_cli.sh` before running the pipeline, so we can remove this message.